src/nni_manager/training_service/common/clusterJobRestServer.ts

```diff
@@ -77,6 +89,26 @@ export abstract class ClusterJobRestServer extends RestServer{
             next();
         });
 
+        router.post(`/version/${this.expId}/:trialId`, (req: Request, res: Response) => {
+            if (this.enableVersionCheck) {
+                try {
+                    if (req.body.tag === 'VCSuccess') {
+                        this.log.info(`Version check in trialKeeper success!`);
+                        this.versionCheckSuccess = true;
+                    } else {
+                        this.errorMessage = req.body.msg;
+                    }
+                } catch(err) {
```

Comment:
Suggest handle the case that different trial jobs report inconsistent results

Reply:
add a logic to handle inconsistent result.